draft-thoji-scone-protocol.md

```diff
@@ -122,19 +132,19 @@ This protocol only works for flows that use the SCONE packet ({{packet}}).
 
 The protocol requires that packets are modified as they transit a
 network element, which provides endpoints strong evidence that the network
-element has the power to drop packets; though see {{security}} for potential
-limitations on this.
+element has the power to apply rate limiting; though see {{security}} for
+potential limitations on this.
 
-The rate limit signal that this protocol carries is independent of congestion
+The throughput advice signal that this protocol carries is independent of congestion
 signals, limited to a single path and UDP packet flow, unidirectional, and
 strictly advisory.
 
 ## Independent of Congestion Signals
 
-Rate limit signals are not a substitute for congestion feedback.  Congestion
+throughput advice signals are not a substitute for congestion feedback.  Congestion
 signals, such as acknowledgments, provide information on loss, delay, or ECN
-markings {{?ECN=RFC3168}} that indicate the real-time condition of a network
-path.  Congestion signals might indicate a throughput that is different from the
+markings {{?ECN=RFC3168}} that suggest the real-time condition of a network
+path.  Congestion signals might suggest a throughput that is different from the
```

Comment:
I think we need a better word than "suggest" for CC signals.

Reply:
Reverted to 'indicate'